assets/js/components/event/RtcPlayer.tsx

```diff
@@ -24,6 +24,14 @@ const RtcPlayer = ({ isMyself, presenter, playerCallbacks, setPresenters }: RtcP
   };
   playerCallbacks[presenter.email] = connectStreams;
 
+  const isSourceDisabled = (sourceType: SourceType) => {
+    const isEnabled = checkTrackIsEnabled(presenter, sourceType);
+    return isEnabled !== undefined && !isEnabled;
```

Comment:
Isn't it equal to `isEnabled === false`?

Reply:
Yeah, it is. I'll change that